src/dmd/cparse.d

```diff
@@ -2402,6 +2437,26 @@ final class CParser(AST) : Parser!AST
         }
     }
 
+    /*************************
+     * Simple __asm__ parser
+     * https://gcc.gnu.org/onlinedocs/gcc/Asm-Labels.html
+     * simple-asm-expr:
+     *   asm ( asm-string-literal )
+     *
+     * asm-string-literal:
+     *   string-literal
+     */
+    private AST.Expression cparseAsmLabel()
+    {
+        nextToken();     // move past __asm__
```

Comment:
The comments alternate between calling it `asm` and `__asm__`. Please use `asm` consistently, and remark somewhere that `__asm` and `__asm__` are remapped to `asm`.

Reply:
#12552